projects/amun/index.js

```diff
@@ -31,11 +53,15 @@ function chainTvl(chain, tokens) {
 }
 
 module.exports = {
+    timetravel: false,
```

Comment:
Will it have some consequences on how DFI and PECO are shown on their UI? Are we going to lose past history of DFI and PECO?

Reply:
probably yes - it will only show the current one